src/styles/style.scss

```diff
@@ -0,0 +1,557 @@
+@import '../styles/mixins';
+
+html { font-family: Cinzel, sans-serif; }
+
+p {
+    margin: 0;
+}
+
+// body 
+.main {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+
+// header 
+.header {
+    background-image: url(../images/header-mobile.png);
+    height: 100vh;
+    background-size: cover;
+    background-position: center;
+    
+
+    @include on-Desktop {
+        background-image: url(../images/header-desktop.png);
+    }
+    &__bar {
+        padding: 25px 20px 129px;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+
+        @include on-Desktop {
+            padding: 50px 120px 83px ;
+        }
+    }
+
+    &__icons {
+        display: flex;
+        gap: 24px;
+
+        @include on-Tablet {
+            gap: 48px;
+        }
+    }
+    
+    &__title {
+        color: #fff;
+        text-align: center;
+        font-size: 40px;
+        font-weight: 700;
+        margin: 0 20px;
+        padding: 23px 40px;
+
+        @include on-Desktop {
+            margin: 0 381px;
+            padding: 0 12px;
+            font-size: 80px;
+        }
+    }
+}
+
+// menu
+.menu {
```

Comment:
You should make menu scrollable if the viewport height is lesser than its [Checklist#12](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Make%20it%20possible%20to%20scroll%20the%20menu%20if%20it%20is%20higher%20than%20the%20viewport)